core/src/main/java/org/akaza/openclinica/dao/managestudy/StudyEventDAO.java

```diff
@@ -1270,7 +1277,7 @@ public boolean isThisRepeatingEventScheduledMoreThanOneTime(int studyId,int sed_
 
         if (it.hasNext()) {
             Integer count = (Integer) ((HashMap) it.next()).get("count");
-            if(count.intValue() >= 1) {
+            if(count.intValue() > 1) {
```

Comment:
please refer to the comment in #3444 and reconcile the changes

Reply:
Now change to ount.intValue() > 0, same logic as #3444.
because count >0 means that we found at least one subject who has been scheduled this repeating event more than once